### PR TITLE
live map for mobile

### PR DESCRIPTION
**Pull request by cprogrammer1994**
This pull request includes a

- [x] New feature

`live-map.py` uses the running `example.py` as the backend.

`live-map.py` is currently for Python 2.7
**Pokemons** are visible on the map, they appear and disappear automatically. (*NEED NO REFRESH*)

- One can click on the Pokemons to see the countdown in seconds. (this should be improved)
- Clicking on multiple Pokemons will only open **1** **InfoWindow**
- Countdown is visible on high DPI devices as well.
- using **setTimeout** and **ajax** (cause only traffic if the browser is active)

### CODE REVIEW

**Comment by Bitti09** [comment]
@cprogrammer1994 
Error:
Google Maps API warning: NoApiKeys https://developers.google.com/maps/documentation/javascript/error-messages#no-api-keys

**Comment by cprogrammer1994** [comment]
The **Google Maps API** key is taken from the `credentials.json`

**Comment by Bitti09** [comment]
Now it works as expected for me :+1:

**Comment by SophisticaSean** [comment]
@cprogrammer1994 I was not able to get this to work, it just serves a webpage with the body being the html text.

File: live-map.py [hunk 1]
```diff
@@ -110,18 +109,24 @@
 			reload();
 		}
 	</script>
-	<script src="https://maps.googleapis.com/maps/api/js?key=%(gmaps_key)s&callback=initMap" async defer></script>
+	<script src="https://maps.googleapis.com/maps/api/js?callback=initMap" async defer></script>
```
Comment (Bitti09) [inline live-map.py]: removing the apikey  ?

Comment (cprogrammer1994) [inline live-map.py]: things messed up when created a new branch for another Pull Request.

**Pull request closed without merging**